packages/components/src/utils/hooks/useA11yListPosition.ts

```diff
@@ -0,0 +1,17 @@
+import { AccessibilityValue } from 'react-native'
+import { useTranslation } from 'react-i18next'
+
+/** Returns list position used for accessibilityValues */
+export function useA11yListPosition(
+  position: number,
+  total: number,
+): AccessibilityValue {
+  const { t } = useTranslation()
+
+  return {
+    text: t('listPosition', {
+      position,
```

Comment:
Should the +1 be baked into the hook? And rename the argument from `position` to `index` to set the expectation it counts from 0.

For user facing counting, the expectation is it'd never start from 0 so seems reasonable to me to roll that into the utility and only correct as a one-off with -1 if for some reason we were working with values that didn't start from 0.